.github/workflows/ci-for-default-ns.yaml

```diff
@@ -67,7 +66,11 @@ jobs:
           kubectl rook_ceph rook status
           kubectl rook_ceph rook status all
           kubectl rook_ceph rook status cephobjectstores
+          kubectl rook_ceph rbd ls replicapool
 
+          # for testing osd purge scale the osd deplyment
+          kubectl --namespace rook-ceph scale deploy/rook-ceph-osd-0 --replicas=0
+          kubectl-rook_ceph rook purge-osd 0
```

Comment:
You may need to sleep a few seconds between scaling down the osd and purging the osd, otherwise the purge will be refused since the osd is not down yet.